terraform/variables.tf

```diff
@@ -42,5 +42,25 @@ variable "num_nodes" {
 variable "qcow2_image" {
   type        = string
   description = "Ubuntu image for VMs - only needed for libvirt provider"
-  default     = "/bionic-server-cloudimg-amd64.img"
+  default     = "/ubuntu-18.04-server-cloudimg-amd64.img"
 }
+
+variable "overlay_cidr" {
+  type        = string
+  description = "CIDR, classless inter-domain routing"
+  #default     = "192.168.122.0/16"
+}
+
+variable "nr_hugepages" {
+  type        = string
+  description = "Number of Huge pages"
+  default     = "1024"
```

Comment:
I'm not sure if we should not use the minimum here that is 512. Given that these scripts are used to create test VMs and at least I create VMs with 4G of mem, taking half of it just for mayastor is maybe too much. Or is there any particular reason why we allocate 2x more than what we need?

Reply:
Fixed https://github.com/openebs/Mayastor/pull/268/commits/5b38dc66d2408d7b84cd517462429a5cd21232c9